--- wayland-1.18.0/doc/man/Makefile.am.orig	2020-02-14 16:57:11.597516684 +0100
+++ wayland-1.18.0/doc/man/Makefile.am	2020-02-14 16:59:24.780128505 +0100
@@ -30,7 +30,7 @@
 XSLTPROC_PROCESS_MAN = \
 	$(AM_V_GEN)$(MKDIR_P) $(dir $@) && \
 	$(XSLTPROC) -o $@ $(XSLTPROC_FLAGS) $(DOCS_STYLESHEET) $< && \
-	$(SED) -i -e 's/^\.so \(.*\)\.\(.\)$$/\.so man\2\/\1\.\2/' $(MANPAGES_ALIASES)
+	$(SED) -i -e 's/^\.so \([^/]*\)\.\(.\)$$/\.so man\2\/\1\.\2/' $(MANPAGES_ALIASES)
 
 %.1: %.xml
 	$(XSLTPROC_PROCESS_MAN)
