--- wayland-1.0.6/doc/man/Makefile.am.orig	2013-02-14 20:47:25.000000000 +0100
+++ wayland-1.0.6/doc/man/Makefile.am	2013-03-29 10:29:29.977545252 +0100
@@ -31,7 +31,7 @@
 XSLTPROC_PROCESS_MAN = \
 	$(AM_V_GEN)$(MKDIR_P) $(dir $@) && \
 	$(XSLTPROC) -o $@ $(XSLTPROC_FLAGS) $(MANPAGES_STYLESHEET) $< && \
-	$(SED) -i -e 's/^\.so \(.*\)\.\(.\)$$/\.so man\2\/\1\.\2/' $(MANPAGES_ALIASES)
+	$(SED) -i -e 's/^\.so \([^/]*\)\.\(.\)$$/\.so man\2\/\1\.\2/' $(MANPAGES_ALIASES)
 
 %.1: %.xml
 	$(XSLTPROC_PROCESS_MAN)
