Build only HTML; PDF requires patched qt4.
--- wayland-1.2.1/doc/publican/Makefile.am.orig	2013-07-29 19:38:05.000000000 +0200
+++ wayland-1.2.1/doc/publican/Makefile.am	2013-09-25 18:22:49.347339417 +0200
@@ -96,7 +96,7 @@
 	$(AM_V_GEN)$(PUBLICAN) rename --name Wayland \
 		--version "$(WAYLAND_VERSION_MAJOR).$(WAYLAND_VERSION_MINOR)" \
 		--config $(builddir)/publican-copy.cfg
-	$(AM_V_GEN)$(PUBLICAN) build --quiet --langs en-US --pdftool fop --formats html,pdf \
+	$(AM_V_GEN)$(PUBLICAN) build --quiet --langs en-US --formats html \
 		--config $(builddir)/publican-copy.cfg
 	@touch Wayland
 
